misc/pytorch_toolkit/chest_xray_screening/tests/test_export.py

```diff
@@ -0,0 +1,46 @@
+import unittest
+import os
+import json
+from chest_xray_screening.utils.download_weights import download_checkpoint
+from chest_xray_screening.utils.exporter import Exporter, OPENVINO_DIR
+
+
```

Comment:
By default, the tests are sorted in the alphabetic order. This means, that `test_export_ir` will be invoked before `test_export_onnx` and it will fail, since onnx model does not exits. 
You can disable this ordering, but unit tests should not depend on the output of the previous tests. Check, if onnx model exists in the `test_export_ir` and if it does not, export it.